### PR TITLE
Modify sample files

### PR DESCRIPTION
**Pull request by riturajj**
Modify the sample XML files to remove the errors/warnings that were already present.
Added extra data/elements in the files and removed some elements.

### CODE REVIEW

File: post_election_sample_feed_precincts.xml [hunk 1]
```diff
@@ -6,162 +6,190 @@
         <BallotName>
           <Text language="en">Barack H. Obama</Text>
         </BallotName>
+        <ExternalIdentifiers><ExternalIdentifier><Type>other</Type><OtherType>stable</OtherType><Value>vageneral-cand-2013-va-obama</Value></ExternalIdentifier></ExternalIdentifiers>
```
[the remark this answers is not in the document]
Comment (riturajj) [reply post_election_sample_feed_precincts.xml]: "va" before each candidate's last name in the stable Id signifies "Virginia" as this feed is for 2013 Virginia General Election. Modified the <FileDate> for the candidate, updated the date 2015 to 2013.

File: post_election_sample_feed_summary.xml [hunk 2]
```diff
@@ -6,162 +6,190 @@
         <BallotName>
           <Text language="en">Barack H. Obama</Text>
         </BallotName>
+        <ExternalIdentifiers><ExternalIdentifier><Type>other</Type><OtherType>stable</OtherType><Value>vageneral-cand-2013-va-obama</Value></ExternalIdentifier></ExternalIdentifiers>
         <FileDate>2015-06-01</FileDate>
```
[the remark this answers is not in the document]
Comment (riturajj) [reply post_election_sample_feed_summary.xml]: Modified the "FileDate" for the candidate, changed the date 2015 to 2013. Not sure about state abbreviation appended as a part of the stable id. This was referred from one of the previous election feeds (BR).

File: post_election_sample_feed_precincts.xml [hunk 3]
```diff
@@ -3169,6 +3199,7 @@
           <Text language="en">Lieutenant Governor of Virginia</Text>
         </BallotTitle>
         <ElectoralDistrictId>ru0002</ElectoralDistrictId>
+        <ExternalIdentifiers><ExternalIdentifier><Type>other</Type><OtherType>stable</OtherType><Value>vageneral-cont-2013-va2-country-us</Value></ExternalIdentifier></ExternalIdentifiers>
         <Name>Lieutenant Governor</Name>
```
[the remark this answers is not in the document]
Comment (riturajj) [reply post_election_sample_feed_precincts.xml]: Modified the stable ids - replaced the *-va-country-us with the respective offices as a part of the stable ids.

File: post_election_sample_feed_summary.xml [hunk 4]
```diff
@@ -571,10 +606,11 @@
           <CandidateIds>can10438</CandidateIds>
         </BallotSelection>
         <BallotTitle>
-          <Text language="en">Member House of Delegates</Text>
+          <Text language="en">Member House of Delegates District - 058</Text>
```
[the remark this answers is not in the document]
Comment (riturajj) [reply post_election_sample_feed_summary.xml]: Added/Modified the District reporting unit Ids in the BallotTitles and Names of the Contests.

**Comment by jdmgoogle** [comment]
Raj,

Looks good. My only recommendation is to run the file through

`xmllint --format`

to make the indentation and formatting consistent before submitting.

**Pull request merged**